### PR TITLE
Glassfish ri test copyright fixes

### PR DESCRIPTION
**Pull request by vinayvishal**

### CODE REVIEW

**Comment by vinayvishal** [comment]
@glassfishrobot Run CI tests please

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/banner.txt [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
- * supplements a celles-ci.  Distribue par des licences qui en 
- * restreignent l'utilisation.
- * 
- * Cette distribution peut comprendre des composants developpes par des 
- * tierces parties. Sun, Sun Microsystems, le logo Sun, Java et J2EE 
- * sont des marques de fabrique ou des marques deposees de Sun 
- * Microsystems, Inc. aux Etats-Unis et dans d'autres pays.
- *
+
+ NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/banner.txt]: What happened to "DO"?  Add it, plus appropriate indentation.

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/checkoutAck.txt [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
- * supplements a celles-ci.  Distribue par des licences qui en 
- * restreignent l'utilisation.
- * 
- * Cette distribution peut comprendre des composants developpes par des 
- * tierces parties. Sun, Sun Microsystems, le logo Sun, Java et J2EE 
- * sont des marques de fabrique ou des marques deposees de Sun 
- * Microsystems, Inc. aux Etats-Unis et dans d'autres pays.
- *
+
+ NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/checkoutAck.txt]: Again, add "DO".

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/checkoutForm.txt [hunk 3]
```diff
@@ -1,32 +1,46 @@
...
- * supplements a celles-ci.  Distribue par des licences qui en 
- * restreignent l'utilisation.
- * 
- * Cette distribution peut comprendre des composants developpes par des 
- * tierces parties. Sun, Sun Microsystems, le logo Sun, Java et J2EE 
- * sont des marques de fabrique ou des marques deposees de Sun 
- * Microsystems, Inc. aux Etats-Unis et dans d'autres pays.
- *'
+
+ NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/checkoutForm.txt]: And again.

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/errorpage.txt [hunk 4]
```diff
@@ -1,31 +1,45 @@
...
- * en vigueur de la FAR (Federal Acquisition Regulations) et des 
- * supplements a celles-ci.  Distribue par des licences qui en 
- * restreignent l'utilisation.
- * 
- * Cette distribution peut comprendre des composants developpes par des 
- * tierces parties. Sun, Sun Microsystems, le logo Sun, Java et J2EE 
- * sont des marques de fabrique ou des marques deposees de Sun 
- * Microsystems, Inc. aux Etats-Unis et dans d'autres pays.
+<%@ page isErrorPage="true" %>
+ NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/errorpage.txt]: Ok, I'm going to assume you'll find all the instances of this problem and fix them all.

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/template/screendefinitions.txt [hunk 5]
```diff
@@ -1,31 +1,45 @@
...
- * en vigueur de la FAR (Federal Acquisition Regulations) et des 
- * supplements a celles-ci.  Distribue par des licences qui en 
- * restreignent l'utilisation.
- * 
- * Cette distribution peut comprendre des composants developpes par des 
- * tierces parties. Sun, Sun Microsystems, le logo Sun, Java et J2EE 
- * sont des marques de fabrique ou des marques deposees de Sun 
- * Microsystems, Inc. aux Etats-Unis et dans d'autres pays.
+<%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>
+ NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/template/screendefinitions.txt]: Shouldn't this be inside a JSP comment, with the taglib declaration following?

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/template/template.txt [hunk 6]
```diff
@@ -1,32 +1,46 @@
-<%--
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/server/web/template/template.txt]: Again, the comment start is being removed.  Why?

File: appserver/tests/appserv-tests/devtests/web/jspBomInClassicSyntax/docroot/UTF-8.jsp [hunk 7]
```diff
@@ -1 +1,43 @@
-﻿this is a test
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/web/jspBomInClassicSyntax/docroot/UTF-8.jsp]: All of these UTF-* files should probably not be modified.  I assume they have very specific UTF-* formatted data.  They're small enough that they don't need a copyright header.

File: appserver/tests/appserv-tests/devtests/web/jspBomInClassicSyntaxEncodingMismatchPageDir/docroot/encodingMismatch.jsp [hunk 8]
```diff
@@ -1 +1,43 @@
-﻿<%@ page pageEncoding="UTF-16BE"%>
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/web/jspBomInClassicSyntaxEncodingMismatchPageDir/docroot/encodingMismatch.jsp]: Again, probably best not to change this file.

File: appserver/tests/appserv-tests/devtests/web/jspTagLibraryValidatorXMLViewStaticInclude/docroot/jsp/a.jsp [hunk 9]
```diff
@@ -1,3 +1,45 @@
...
+    recipient has the option to distribute your version of this file under
+    either the CDDL, the GPL Version 2 or to extend the choice of license to
+    its licensees as provided above.  However, if you add GPL Version 2 code
+    and therefore, elected the GPL Version 2 license, then the option applies
+    only if the new code is made subject to such option by the copyright
+    holder.
+
+--%>
+
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/web/jspTagLibraryValidatorXMLViewStaticInclude/docroot/jsp/a.jsp]: Does this work correctly with the JSP comment above the XML declaration?

File: appserver/tests/appserv-tests/devtests/web/servlet-3.0/serveJspAndStaticResourceFromLocalJar/resources/abc.jsp [hunk 10]
```diff
@@ -1 +1,43 @@
...
+    recipient has the option to distribute your version of this file under
+    either the CDDL, the GPL Version 2 or to extend the choice of license to
+    its licensees as provided above.  However, if you add GPL Version 2 code
+    and therefore, elected the GPL Version 2 license, then the option applies
+    only if the new code is made subject to such option by the copyright
+    holder.
+
+--%>
+
 Hello World!
```
Comment (bshannon) [inline appserver/tests/appserv-tests/devtests/web/servlet-3.0/serveJspAndStaticResourceFromLocalJar/resources/abc.jsp]: Adding the copyright header to these trivial files seems gratuitous.
And I'm worried that these test won't work correctly after these changes.

File: appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/application.js [hunk 11]
```diff
@@ -140,6 +140,6 @@ var rules = {
         elem.onclick = app.post;
     }
 };
-Behaviour.addLoadEvent(app.initialize);
-Behaviour.register(rules);
+// Behaviour.addLoadEvent(app.initialize);
+// Behaviour.register(rules);
```
Comment (bshannon) [inline appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/application.js]: Why is this being commented out?

**Review by bshannon — CHANGES_REQUESTED** [review]
Lots of things to fix; see comments.

File: appserver/tests/appserv-tests/devtests/web/jspTagLibraryValidatorXMLViewStaticInclude/docroot/jsp/a.jsp (hunk 9 above)
Comment (vinayvishal) [reply appserver/tests/appserv-tests/devtests/web/jspTagLibraryValidatorXMLViewStaticInclude/docroot/jsp/a.jsp]: Originally file had the license after XML declaration, copyright plugin treats this a case of file without any license and puts the header at the top. I have reverted and have retained the original file.

File: appserver/tests/appserv-tests/devtests/web/servlet-3.0/serveJspAndStaticResourceFromLocalJar/resources/abc.jsp (hunk 10 above)
Comment (vinayvishal) [reply appserver/tests/appserv-tests/devtests/web/servlet-3.0/serveJspAndStaticResourceFromLocalJar/resources/abc.jsp]: Reverted.

File: appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/application.js (hunk 11 above)
Comment (vinayvishal) [reply appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/application.js]: I missed cleaning it up. This is with regards to comment on Row 12 where you suggested to try to remove the non essential piece of code from these files. To that effect, appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/behaviour.js too has been removed in the updated PR. The test works fine even after these removals.

File: appserver/tests/v2-tests/appserv-tests/devtests/classloader/servlet/docroot/tags/shipDate.tag [hunk 12]
```diff
@@ -1,30 +1,45 @@
-<%--
```
[the remark this answers is not in the document]
Comment (vinayvishal) [reply appserver/tests/v2-tests/appserv-tests/devtests/classloader/servlet/docroot/tags/shipDate.tag]: Fixed.

File: appserver/tests/appserv-tests/devtests/web/jspBomInClassicSyntax/docroot/UTF-8.jsp (hunk 7 above)
Comment (vinayvishal) [reply appserver/tests/appserv-tests/devtests/web/jspBomInClassicSyntax/docroot/UTF-8.jsp]: Reverted.

File: appserver/tests/appserv-tests/devtests/web/jspTagLibraryValidatorXMLViewStaticInclude/docroot/jsp/a.jsp (hunk 9 above)
Comment (bshannon) [reply appserver/tests/appserv-tests/devtests/web/jspTagLibraryValidatorXMLViewStaticInclude/docroot/jsp/a.jsp]: I think the problem is that it's an XML format JSP file, which would normally be named with a ".jspx" extension.  Is it possible to rename the file?

File: appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/application.js (hunk 11 above)
Comment (bshannon) [reply appserver/tests/embedded/maven-plugin/websockets/src/main/webapp/javascripts/application.js]: If the unneeded dependency has been removed, these lines can just be removed.

**Review by bshannon — DISMISSED** [review]

**Comment by vinayvishal** [comment]
@glassfishrobot Run CI tests please

**Comment by vinayvishal** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]

**Pull request merged**